ArduPlane/Log.cpp

```diff
@@ -298,20 +303,22 @@ const struct LogStructure Plane::log_structure[] = {
 // @LoggerMessage: CTUN
 // @Description: Control Tuning information
 // @Field: TimeUS: Time since system startup
-// @Field: NavRoll: desired roll
-// @Field: Roll: achieved roll
-// @Field: NavPitch: desired pitch
-// @Field: Pitch: achieved pitch
+// @Field: NavRll: desired roll
+// @Field: Rll: achieved roll
+// @Field: NavPit: desired pitch
+// @Field: Pit: achieved pitch
```

Comment:
If were allowed to change the names (which might break log review tools) I vote we just remove them (which will break log review tools). They in `ATT` in anycase. `ThO`, `RdrOut` are also in `AETR`....

One for the dev call.

Reply:
Why would you want to do this? All it seems to be doing is making logs less user friendly (which they are already pretty bad). Is there any advantage that's worth it?

Reply:
You get 64 chars total for all the labels, so sometimes to add more your have to shorten the existing ones.

Reply:
Got it.